### DISCUSSION TITLE
Getting the names of people in your instance

### DISCUSSION THREAD

**Discussion by me-sideways** [vrcx-team/VRCX#354] [General]
Hi everyone,

Im working on a project, and part of that is getting the names of people in your instance 

Someone suggested to look at VRCX and here I am!

Im not at expert coding, and a little lost, is there some where I can understand how VRCX generates a list of people in the current users instance?

**Comment by Natsumi-sama** [comment]
VRCX reads the games log files located here `%AppData%/../LocalLow/VRChat/VRChat/output_log_*.txt`. But that's the easy part the hard part is reading mesh shape keys maybe you could do it with some cursed shader magic but unlikely it seems like what you are aiming for is only possible as a mod in which case you wouldn't need to read the log file.

**Reply by me-sideways** [reply]
Thanks you, what I need is join and leave notification, it looks like those logs do that!
